tests/test_main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+from app.main import get_human_age
+
```

Comment:
Please, add tests for each class of equivalents:
* when age is under 15
* when age is between 15 and 24
* when age is greater than 24

Reply:
could you add tests for `24` and `25` values

Reply:
It's important to add tests with values between 15 and 28.
Please, check also all boundary values